packages/features/eventtypes/components/tabs/recurring/RecurringEventController.tsx

```diff
@@ -2,22 +2,44 @@ import { useState } from "react";
 import { useFormContext } from "react-hook-form";
 
 import useLockedFieldsManager from "@calcom/features/ee/managed-event-types/hooks/useLockedFieldsManager";
-import type { EventTypeSetup } from "@calcom/features/eventtypes/lib/types";
+import type {
+  EventTypeSetup,
+  InputClassNames,
+  SelectClassNames,
+  SettingsToggleClassNames,
+} from "@calcom/features/eventtypes/lib/types";
 import type { FormValues } from "@calcom/features/eventtypes/lib/types";
 import { classNames } from "@calcom/lib";
 import { useLocale } from "@calcom/lib/hooks/useLocale";
 import { Frequency } from "@calcom/prisma/zod-utils";
 import type { RecurringEvent } from "@calcom/types/Calendar";
 import { Alert, Select, SettingsToggle, TextField } from "@calcom/ui";
 
-type RecurringEventControllerProps = {
+export type RecurringEventControllerProps = {
   eventType: EventTypeSetup;
   paymentEnabled: boolean;
+  customClassNames?: EventRecurringTabCustomClassNames;
+};
+
+export type EventRecurringTabCustomClassNames = {
+  container?: string;
+  recurringToggle?: SettingsToggleClassNames;
```

Comment:
Not sure about `recurringToggle` name. First, within it we have `switchContainer` and we should pick either we use "toggle" or "switch" in both - use 1 word to describe the same concept. Second, in other classnames we simply use `container` instead of `switchContainer`. I would rename `switchContainer` to `container` and `recurringToggle` to something like `recurringEventHeader` not 100% sure, because we can't style the toggle itself, but label and description are not children of toggle but rather its equals.

Reply:
good idea, will make changes